scenes/menu/char_select.tscn

```diff
@@ -293,6 +295,30 @@ layout_mode = 2
 char_id = "random_s5"
 portrait_texture = ExtResource("29_7gojn")
 
+[node name="SFCharacterSelect" type="PanelContainer" parent="CenterContainer"]
+layout_mode = 2
+mouse_filter = 2
+theme_override_styles/panel = SubResource("StyleBoxEmpty_8u80i")
+
+[node name="Rows" type="VBoxContainer" parent="CenterContainer/SFCharacterSelect"]
+layout_mode = 2
+mouse_filter = 2
+alignment = 1
+
+[node name="Row1" type="HBoxContainer" parent="CenterContainer/SFCharacterSelect/Rows"]
+layout_mode = 2
+alignment = 1
+
+[node name="Ryu" parent="CenterContainer/SFCharacterSelect/Rows/Row1" instance=ExtResource("6_3lmwy")]
+layout_mode = 2
+char_id = "ryu"
+portrait_texture = ExtResource("30_mommm")
+
+[node name="RandomSF" parent="CenterContainer/SFCharacterSelect/Rows/Row1" instance=ExtResource("6_3lmwy")]
```

Comment:
If you're bored, you can also add a random all seasons button.
I don't know the best way to put that in the UI though.
Someone was requesting it.